python/ray/data/_internal/execution/interfaces.py

```diff
@@ -216,6 +217,9 @@ class TaskContext:
     # operator.
     task_idx: int
 
+    # The iterator of sub progress bar that task is responsible to update.
```

Comment:
Should the `sub_progress_bar_iter` be a dict? Being an iterator looks implicit that all sub-operators/stages are used in order.